tests/test_flow.py

```diff
@@ -2830,6 +2830,35 @@ def test_write_to_parquet_partition_by_datetime(tmpdir):
     assert read_back_df.equals(expected_df)
 
 
+@pytest.mark.parametrize("max_events", [1, 5])
+def test_write_to_single_partition_parquet(tmpdir, max_events):
```

Comment:
I guess this test should check the file name?

It would be good for it to also run the flow twice to make sure the second run overwrites the first run. (this should work without rebuilding the flow – just terminate and rerun)

Reply:
done